ctdcal/odf_io.py

```diff
@@ -2,6 +2,9 @@
 import numpy as np
 import datetime as dt
 import gsw
+import csv
+from collections import OrderedDict
```

Comment:
If the minimum supported python version is 3.7, this can be a regular builtin dict.

Reply:
I saw that while implementing the relevant chunk of code and wasn't sure if we had a minimum python version.. so I went with Ordered over regular

Reply:
At some point, a policy should probably be implemented, you'll want to closely track your dependancies, e.g. pandas at the time of my writing this comment supports: "Officially Python 3.6.1 and above, 3.7, and 3.8." Numpy looks like this: https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table